components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/SubscriptionValidationDAO.java

```diff
@@ -1187,7 +1224,6 @@ public API getAPIByContextAndVersion(String context, String version, String depl
                         } else {
                             api.setPolicy(null);
                         }
-                        return api;
```

Comment:
Seems this change is not required.

Reply:
done with https://github.com/wso2/carbon-apimgt/pull/12168/commits/27e7b0edcdab808f7a4efe9b2a366368cfca0652